src/commands/guide/get.ts

```diff
@@ -0,0 +1,170 @@
+import { Args, Flags, ux } from "@oclif/core";
+
+import * as ApiV1 from "@/lib/api-v1";
+import BaseCommand from "@/lib/base-command";
+import { formatDateTime } from "@/lib/helpers/date";
+import { ApiError } from "@/lib/helpers/error";
+import { formatErrorRespMessage, isSuccessResp } from "@/lib/helpers/request";
+import { spinner } from "@/lib/helpers/ux";
+import { formatConditions } from "@/lib/marshal/conditions";
+
+const formatStep = (step: ApiV1.GetGuideResp["steps"][number]) => {
+  return `${step.schema_key} (${step.schema_variant_key})`;
+};
+
+export default class GuideGet extends BaseCommand<typeof GuideGet> {
+  static summary = "Display a single guide from an environment.";
+
+  static flags = {
+    environment: Flags.string({
+      default: "development",
+      summary: "The environment to use.",
+    }),
+    "hide-uncommitted-changes": Flags.boolean({
+      summary: "Hide any uncommitted changes.",
+    }),
+  };
+
+  static args = {
+    guideKey: Args.string({
+      required: true,
+    }),
+  };
+
+  static enableJsonFlag = true;
+
+  async run(): Promise<ApiV1.GetGuideResp | void> {
+    spinner.start("‣ Loading");
+
+    const { guide } = await this.loadGuide();
+
+    spinner.stop();
+
+    const { flags } = this.props;
+    if (flags.json) return guide;
+
+    this.render(guide);
+  }
+
+  private async loadGuide(): Promise<{
+    guide: ApiV1.GetGuideResp;
+  }> {
+    const guideResp = await this.apiV1.getGuide(this.props);
+
+    if (!isSuccessResp(guideResp)) {
+      const message = formatErrorRespMessage(guideResp);
+      ux.error(new ApiError(message));
+    }
+
+    return {
+      guide: guideResp.data,
+    };
+  }
+
+  render(guide: ApiV1.GetGuideResp): void {
+    const { guideKey } = this.props.args;
+    const { environment: env, "hide-uncommitted-changes": committedOnly } =
+      this.props.flags;
+
+    const qualifier =
+      env === "development" && !committedOnly ? "(including uncommitted)" : "";
+
+    this.log(
+      `‣ Showing guide \`${guideKey}\` in \`${env}\` environment ${qualifier}\n`,
+    );
+
+    /*
+     * Guide table
+     */
+
+    const getStatusValue = () => {
+      const baseStatus = guide.active ? "Active" : "Inactive";
+
+      if (guide.active_from || guide.active_until) {
+        const fromText = guide.active_from
+          ? `from ${guide.active_from}`
+          : "immediately";
+        const untilText = guide.active_until
+          ? `until ${guide.active_until}`
+          : "with no end time";
+        return `${baseStatus} (${fromText} ${untilText})`;
+      }
+
+      return baseStatus;
+    };
+
+    const formatActivationRulesValue = () => {
```

Comment:
Nit: I would also just pull this and other formatter funcs above/at the top out into helpers.ts.